src/components/controls/gamepad-controls.js

```diff
@@ -0,0 +1,110 @@
+var registerControls = require('../../core/controls').registerControls;
```

Comment:
It would admittedly be weird to have locomotion by default on mobile but not on desktop, and yeah most of the time you probably want the cardboard button for interacting with things, not moving. So i guess we're not including any default locomotion then – maybe hotspot navigation later on.